plugins/src/terminal/mod.rs

```diff
@@ -116,10 +116,8 @@ fn detect_terminal() -> (PathBuf, &'static str) {
     const SYMLINK: &str = "/usr/bin/x-terminal-emulator";
 
     if let Ok(found) = read_link(SYMLINK) {
-        if let Ok(found) = read_link(&found) {
             return (found, "-e");
-        }
```

Comment:
Just updated, but I don't understand why we need to attempt resolving the symlink twice here ?

Reply:
That's just how Debian systems have this symlink configured, because of their use of `/etc/alternatives`. `/usr/bin/x-terminal-emulator` is a symlink to `/etc/alternatives/x-terminal-emulator`, which is a symlink to whichever binary was designated as the alternative option for that.